_infra/helm/case/values.yaml

```diff
@@ -85,3 +85,7 @@ crons:
     name: case-action-event-retry-scheduler
     cron: "0 * * * *"
     target: "retry-event"
+
+surveySvc:
+  multipleFormTypeSupportedSurveysIds: "202"
```

Comment:
2 things:
- multipleFormTypeSupportedSurveysIds should be multipleFormTypeSupportedSurveyIds, as it's more grammatically correct.  If you take on this change, you'll have to make the change everywhere else that uses that config item name!
- Is there a reason why this one is only 202 and the one in application.yml has 202 and 203?

Reply:
No reason, just a guidance that it's comma separated . With respect to naming I will change it.